src/main/java/com/google/jenkins/flakyTestHandler/plugin/FlakyTestResultAction.java

```diff
@@ -75,7 +75,7 @@
    */
   public FlakyTestResultAction(AbstractBuild build, BuildListener listener) {
     this.build = build;
-    AbstractTestResultAction action = build.getTestResultAction();
+    AbstractTestResultAction action = build.getAction(AbstractTestResultAction.class);
```

Comment:
Will this work if there are multiple TestResultActions - aka cucumber plugin and flaky test?